src/helpers/urlHelper.js

```diff
@@ -0,0 +1,5 @@
+import urlJoin from 'url-join';
+export const buildUrl = (...uris) => {
```

Comment:
This can be cleaned up a bit:

```
const baseUrl = process.env.REACT_APP_SERVICE_BASE_URL;

const buildUrl = (...uris) =>  urlJoin(baseUrl, ...uris);
```

Reply:
Thanks for the thorough review. I applied your suggested changes but am looking at an error right now:

```
Uncaught TypeError: a is not a function
    at InQueueCallbackComponent.cbCallButtonAccessiblity (bundle.js:2)
    at InQueueCallbackComponent.startCall (bundle.js:2)
    at onClick (bundle.js:2)
    at Object.u (twilio-flex.min.js:1546)
    at h (twilio-flex.min.js:1546)
    at twilio-flex.min.js:1546
    at E (twilio-flex.min.js:1546)
    at A (twilio-flex.min.js:1546)
    at I (twilio-flex.min.js:1546)
    at O (twilio-flex.min.js:1546)
  ```
  
  Does using `const` remove the need to export the function?

Reply:
I wonder maybe we want `const baseUrl = 'process.env.REACT_APP_SERVICE_BASE_URL';` but if that doesn't work either, let's just keep what we have.

Reply:
Yeah, let's keep what we have since that actually works. 😅